README.md

```diff
@@ -1,70 +1,84 @@
 # KOSMOS
 
-Kosmos是移动云开源的分布式云原生联邦集群技术的集合，其名称kosmos：k代表**k**ubernetes，c**osmos**表示宇宙（希腊语），寓意kubernetes的无限扩展。目前，kosmos主要包括三大模块，分别是：**多集群网络**、**多集群管理编排**、**多集群调度**。此外，kosmos还配备一款kosmosctl工具， 可以快速进行kosmos组件部署、添加集群、测试网络连通性等工作。
+Kosmos is an open-source collection of distributed cloud-native federated cluster technologies. The name "kosmos" combines 'k' representing Kubernetes and 'cosmos' which means universe in Greek, symbolizing the limitless expansion of Kubernetes. Currently, Kosmos primarily consists of three major modules: ClusterLink, ClusterTree and Scheduler. Additionally, Kosmos is equipped with a tool called kosmosctl, which allows for quick deployment of Kosmos components, adding clusters, and testing network connectivity.
```

Comment:
May be this is better, What do you think?
Kosmos is an open-source, all-in-one distributed cloud-native solution.

@wuyingjun-lucky @xyz2277  @duanmengkk @wangyizhi1

Reply:
It seems right

Reply:
I have got it